src/convertToObject.js

```diff
@@ -12,7 +12,11 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const result = sourceString.split(';')
+    .map(cssRule => cssRule.split(':').map(cssRuleItem => cssRuleItem.trim()))
+    .filter(cssRule => cssRule.length > 1);
```

Comment:
Можливо так краще буде, на майбутнє
```suggestion
  const result = sourceString.split(';')
    .map(cssRule => cssRule.split(':')
      .map(cssRuleItem => cssRuleItem.trim()))
    .filter(cssRule => cssRule.length > 1);
```